service-base/src/main/java/org/eclipse/hono/service/credentials/BaseCredentialsService.java

```diff
@@ -232,6 +232,18 @@ private void processCredentialsMessageRemoveOperation(final Message<JsonObject>
         }
     }
 
+    /**
+     * Override the following method to extend RegistrationService with implementation
```

Comment:
we are talking about the Credentials service, aren't we?

Reply:
yes, I will correct it.